wsit/boms/bom-ext/pom.xml

```diff
@@ -28,8 +28,8 @@
     <description>Metro Web Services Stack Dependency POM for Metro-CS</description>
 
     <properties>
-        <connector-api.version>1.7.4</connector-api.version>
-        <transaction-api.version>1.3.3</transaction-api.version>
+        <connector-api.version>2.0.0</connector-api.version>
```

Comment:
Is there really 2.0.0 available? I though there's only 2.0.0-RC1...

Reply:
I found it here : https://github.com/eclipse-ee4j/jca-api/releases

Reply:
that does not mean it exists in maven central or other maven repository. We should rely on central and use other repos only temporarily.

To find out latest updates to dependencies the project sees, you can use sth like:
```
mvn -U -C versions:display-dependency-updates -Poss-release,staging
mvn -U -C versions:display-property-updates -Poss-release,staging
```